packages/app/src/components/nominationBanner/editOrSaveButton.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styles from '../nominationInfo/newstyles.module.css';
+
+// Button Edit or Save depending on mode ('view' or 'edit')
+const EditOrSaveButton = props => {
```

Comment:
Since this is only used in one place I think it'd be cleaner to have separate components for edit and save and nominationBanner.js can just use it like `{mode == 'view' ? <ViewButton/> : <EditButton/>}` or something like that.